### ISSUE TITLE
[BUG] Project Does Not Exist

### ISSUE THREAD

**Issue by GarrettBrown-dev** [Luke-zhang-04/processing-vscode#3]
I seem to be running into an issue where, when hitting the play button, the terminal says my project doesn't exist. I've double and triple-checked my path to the Processing folder and nothing seems to be working. I also made sure to add the tasks.json file, and only open the folder I'm working in, but each time I hit the play button, it simply gives me this: 

$ processing-java --sketch=c:\Users\Gman9\ProcessingStuff\NoC_RWalker --run
c:UsersGman9ProcessingStuffNoC_RWalker does not exist.

And then doesn't run anything. Any ideas on why this might be occurring?

*I am on Windows 10

**Comment by Luke-zhang-04** [comment]
Ok, I've never programmed on Windows, so I'm not 100% sure, but my guess is that \ isn't registering as a path separator or something like that. I'll see if I can fix it. BTW what terminal are you using is it CMD?

**Comment by Luke-zhang-04** [comment]
can you try manually running `processing-java --sketch="c:\Users\Gman9\ProcessingStuff\NoC_RWalker" --run` to see if that works?

**Comment by GarrettBrown-dev** [comment]
> can you try manually running `processing-java --sketch="c:\Users\Gman9\ProcessingStuff\NoC_RWalker" --run` to see if that works?

Running it manually does work!! Thank you!! And I'm using the VSCode terminal with Git Bash as the default! It seems to be an issue with the button itself, as when I used the manual command, it uses the actual processing-java terminal, whereas the button swaps to a new terminal simply called "Processing." I also noticed when trying to run it that there aren't any quotes after processing-java. Could that be causing an issue? I don't know if this is relevant, but hopefully it helps!
